client/flutter/lib/pages/travel_advice.dart

```diff
@@ -1,12 +1,11 @@
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
```

Comment:
Are we using this instead of the built-in Material icons because we're moving away from Material? Otherwise, we can use the built-in `check_circle` and `warning` icons

Reply:
Cupertino doesn't even have a warninng icon sadly.
FontAwesome looks far better than Material:
FontAwesome:
<img width="642" alt="Screen Shot 2020-04-26 at 2 34 22 PM" src="https://user-images.githubusercontent.com/11729521/80320521-c2879e00-87cb-11ea-931d-5a1c7915a21b.png">



Material:

<img width="319" alt="Screen Shot 2020-04-26 at 2 38 00 PM" src="https://user-images.githubusercontent.com/11729521/80320533-d206e700-87cb-11ea-94e6-43c722cad7a1.png">